clients/v1/clients.proto

```diff
@@ -33,6 +33,18 @@ service UsersService {
   rpc GetUsers(GetUsersRequest) returns (GetUsersResponse) {
     option (sams_required_scopes) = "profile";
   };
+  // CreateUser creates a new SAMS user with the given email.
+  // Required scopes: sams::user::write
+  //
+  rpc CreateUser(CreateUserRequest) returns (CreateUserResponse) {
+    option (sams_required_scopes) = "sams::user::write";
+  };
+  // DeleteUser deletes a SAMS user with the given email.
```

Comment:
"[...] the given **id**"

However, I suggest not being prescriptive here and just say "[...] deletes the user" so it's not easily fall out of sync.